apps/site/sentry.client.config.ts

```diff
@@ -1,14 +1,30 @@
 // Context: https://docs.sentry.io/platforms/javascript/guides/nextjs/
 
 import * as Sentry from "@sentry/nextjs";
-import type { NextjsOptions } from "@sentry/nextjs/types/utils/nextjsOptions";
+import { Replay } from "@sentry/replay";
 
 const dsn = process.env.NEXT_PUBLIC_SENTRY_DSN;
 
-export const sentryConfig: NextjsOptions = {
+const replaysSamplingRate = Number.parseInt(
+  process.env.NEXT_PUBLIC_SENTRY_REPLAYS_SAMPLING_RATE || "0",
+  10,
+);
+
+Sentry.init({
   dsn,
   enabled: !!dsn,
   environment: process.env.NEXT_PUBLIC_VERCEL_ENV ?? "unset",
-};
-
-Sentry.init(sentryConfig);
+  integrations:
+    replaysSamplingRate > 0 &&
```

Comment:
@kachkaev Ditto comments from https://github.com/hashintel/hash/pull/1232#discussion_r1001140022 here

Reply:
This is being addressed in https://github.com/blockprotocol/blockprotocol/pull/698